internal/provider/resource_monitor.go

```diff
@@ -283,65 +281,16 @@ var monitorSchema = map[string]*schema.Schema{
 		Computed:    true,
 		Sensitive:   true,
 	},
-	"checks_version": {
-		Description: strings.ReplaceAll(`Valid values:
-
-    **v1** Proxy-based infrastructure. We use proxies around the world to make regional checks.
 
-    **v2** Edge-based infrastructure. More advanced infrastructure, allows running low level checks in regions.`, "**", "`"),
-		Type:     schema.TypeString,
-		Optional: true,
-		Computed: true,
-		ValidateDiagFunc: func(v interface{}, path cty.Path) diag.Diagnostics {
-			if v == nil {
-				return nil
-			}
-			s := v.(string)
-			for _, checksVersion := range checksVersions {
-				if s == checksVersion {
-					return nil
-				}
-			}
-			return diag.Diagnostics{
-				diag.Diagnostic{
-					AttributePath: path,
-					Severity:      diag.Error,
-					Summary:       `Invalid "checks_version"`,
-					Detail:        fmt.Sprintf("Expected one of %v or nil", checksVersions),
-				},
-			}
-		},
-	},
 	"ip_version": {
 		Description: strings.ReplaceAll(`Valid values:
 
     **ipv4** Use IPv4 only,
 
-    **ipv6** Use IPv6 only
-
-    Note: ip_version is used only if "checks_version" is set to "v2".`, "**", "`"),
+    **ipv6** Use IPv6 only`, "**", "`"),
 		Type:     schema.TypeString,
 		Optional: true,
 		Computed: true,
-		ValidateDiagFunc: func(v interface{}, path cty.Path) diag.Diagnostics {
-			if v == nil {
-				return nil
-			}
-			s := v.(string)
-			for _, checksVersion := range ipVersions {
-				if s == checksVersion {
-					return nil
-				}
-			}
-			return diag.Diagnostics{
-				diag.Diagnostic{
-					AttributePath: path,
-					Severity:      diag.Error,
-					Summary:       `Invalid "ip_version"`,
-					Detail:        fmt.Sprintf("Expected one of %v or nil", ipVersions),
-				},
-			}
-		},
```

Comment:
Why do you want to remove the IP version validation? If that was an accident, please revert 🙏

Reply:
Right, took that one when I saw `checksVersion` which I assume is a typo; will revert, thx!

I did run `mage gen`, and `make fmt` and the others too 👀 will rerun as well